### PR TITLE
pkp/pkp-lib#412 possibility to define the size of cover thumbnails

### PR DESCRIPTION
**Pull request by bozana**
I moved the code to master and separated the system specific code... :-)
The other pull request can be removed... :-)

### CODE REVIEW

File: templates/controllers/tab/settings/appearance/form/appearanceForm.tpl [hunk 1]
```diff
@@ -21,4 +21,17 @@
...
+		{/fbvFormSection}
+		{fbvFormSection}
+			{fbvElement type="text" id="coverThumbnailsMaxHeight" value=$coverThumbnailsMaxHeight size=$fbvStyles.size.SMALL label="manager.setup.coverThumbnailsMaxHeight" required="true"}
+		{/fbvFormSection}
+		{fbvFormSection list="true"}
+			{fbvElement type="checkbox" label="manager.setup.coverThumbnailsResize" id="coverThumbnailsResize" checked=$coverThumbnailsResize}
+		{/fbvFormSection}
+	{/fbvFormArea}
+{/capture}
 {include file="core:controllers/tab/settings/appearance/form/appearanceForm.tpl" newContentFormContent=$newContentFormContent featuredContentFormContent=$featuredContentFormContent}
```
Comment (asmecher) [inline templates/controllers/tab/settings/appearance/form/appearanceForm.tpl]: It's not strictly necessary, but it's a good practice to add the variables this template will use to the include call as a hint to developers:

    thumbnailsSizeSettings=$thumbnailsSizeSettings

File: plugins/generic/tinymce [hunk 2]
```diff
@@ -1 +0,0 @@
-Subproject commit 949badbccc42f10d732b71df7d0fb1e9b585b565
```
Comment (asmecher) [inline plugins/generic/tinymce]: Is this here intentionally?

Comment (bozana) [inline plugins/generic/tinymce]: oh, no it's not :-( how to proceed? to change and make a new pull request?
THANKS A LOT!!!

Comment (bozana) [inline plugins/generic/tinymce]: ah no, I will push everything again but in the same branch i.e. in the same pull request...

**Pull request merged**